INSTALL.rst

```diff
@@ -247,9 +247,9 @@ venv
   pip install -r requirements.txt
   # Install build dependencies of scikit-image
   pip install -r requirements/build.txt
-  # Build scikit-image from source
-  spin build
-  # The new version lives under `${PWD}/build-install/.../site-packages`.
+  # Install scikit-image in editable mode, which will make sure that
+  # scikit-image is recompiled if necessary on import
+  spin install -v
```

Comment:
I'd split the flow here between everything up to install, and then, separately, show a few examples of what you can do with it.

Also, match the conda version to what is recommended here.

Remove mentions of Anaconda, we should only recommend miniforge.